app/members/tracy-chen.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.njk' %}
+
+{% block title %}Tracy Chen | Bitcoin Cash Node{% endblock %}
+{% block description %}{% endblock %}
+
+{% block body %}
+    <section class="bg-dark">
+      <div class="container py-5">
+        <div class="row py-5 text-center text-md-left justify-content-center">
+          <div class="col-md-4 text-md-right">
+            <img class="rounded" src="/static/img/team/tracy_chen.jpg" alt="Tracy" width="200">
+          </div>
+          <div class="col-md-6 mt-5">
+            <h1 class="text-white font-weight-bold">Tracy Chen</h1>
+            <h4 class="text-white">BCHN Representative</h4>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="bg-light">
+      <div class="container py-5">
+        <div class="row my-5 justify-content-center">
+          <div class="col-md-9 text-lg">
+            <p>Tracy is the representative of BCHN, whose main role is
+            to establish contact channels and maintain communication on
+            behalf of BCHN team, BCH users and all ecological parties
+            (such as the exchange, mining pool, and wallet) to ensure
+            BCHN’s decisions to be fully approved and verified. She also
+            runs BCHN's community and SNS in China to update the latest
+            project progress and financial reports.
+            She used to be a traditional media professional; and has
+            begun to join the encryption industry since 2018. She has
```

Comment:
She has worked as a traditional media professional and entered the cryptocurrency industry in 2018.
She has...

Reply:
Thanks, updated